### PR TITLE
docs: Describe patch submission workflow

### LINKED ISSUE #101: Git workflow description in Developers' Manual
Currently there is no description of the git work flow in the Developers' Manual. As a consequence it is not clear to the potential patch submitter where they should branch from.

The work flow used in the XCSoar project is relatively simple. Despite it needs to be documented.

### PR DESCRIPTION
**Pull request by Exadios**
This may be a bit wordy and I'm not sure that I've used the correct terminology. In any case have a look and see whether its clear.

Closes #101

### CODE REVIEW

File: doc/manual/en/XCSoar-developer-manual.tex [hunk 1]
```diff
@@ -74,11 +74,59 @@ \chapter{Introduction}\label{cha:introduction}
...
-Submit patches or \texttt{git pull} requests to the developer mailing
-list (\texttt{xcsoar-devel@lists.sourceforge.net}) or at
-\href{https://github.com/XCSoar/XCSoar}{our GitHub repository}.
+There are two methods to summit patches to the upstream XCSoar repository.
+
+The XCSoar repository is currently hosted at
+\href{https://github.com/XCSoar/XCSoar}{XCSoar GitHub repository}.
+GitHub provides a method for submitting patches using their Web interface.
+The easiest way to use this method is to make another XCSoar repository by
+ 'forkng' from 'XCSoar/XCSoar'. Make sure that your local repository
```
Comment (Turbo87) [inline doc/manual/en/XCSoar-developer-manual.tex]: typo :wink:

Comment (Exadios) [reply doc/manual/en/XCSoar-developer-manual.tex]: But I really did mean to say "forkng". :)

Anyway I've made the change and pushed it to the repository from which I made the pull request.

For my info does this trigger some sort of notification at your end? Can you see the updated branch or do I need to start a new pull request?

Comment (Turbo87) [reply doc/manual/en/XCSoar-developer-manual.tex]: > For my info does this trigger some sort of notification at your end?

yes

> Can you see the updated branch

yes, you should be able to see f86ecab being mentioned below this comment

> do I need to start a new pull request?

no

Comment (Exadios) [reply doc/manual/en/XCSoar-developer-manual.tex]: I cannot do anything about the German, French, Polish and Brazilian versions of the manual. How is that usually handled?

Comment (Turbo87) [reply doc/manual/en/XCSoar-developer-manual.tex]: I have no idea... but I think the developer manual is only available in English anyway, right?

Comment (Exadios) [reply doc/manual/en/XCSoar-developer-manual.tex]: Great. This seems to be the most efficient method of submitting patches.

**Comment by Exadios** [comment]
@lordfolken sure. I'll do it next time I'm down at the club.
